week-2-practise/level-2/6-map.js

```diff
@@ -3,8 +3,13 @@
 // 2. Declare a new array, `percentagesFormatted`, containing
 //    - each item in `percentages` formatted by your function
 
+function formatPercentage(number){
+  return number+="%";
```

Comment:
The `.map()` method creates a **new** array from the old array transforming it based on the callback. That means we don't need to update the `number` variable here. We are only using it to calculate the new string value in the new array.

That means the `+=` operator is unnecessary. You can just use the `+` operator.